### PR TITLE
Update block templates to match coding standards

### PR DESCRIPTION
**Pull request by tarecord**
After generating some blocks recently, I noticed that the templates do not match our current coding standards.

### CODE REVIEW

**Review by nickpelton — APPROVED** [review]
Heck Yeah! 🚀 

Note, we'll need to do a release to ship this. I'll plan on doing it in the new year.

**Review by ChrisMKindred — APPROVED** [review]
🌮

**Comment by defunctl** [comment]
@tarecord Thanks so much for doing this!

I started a v2 conversion a long time ago but never got to finish it. 

There's a few more you could include for this release if you have time:

1. https://github.com/moderntribe/tribe-libs/commit/9fabe2c566c1a506116e9e89c72322c37ade8b5b#diff-c4b17a47324748d068e45da15eb798ea25fcc161d1ffe2835aa6629e76a77dd3
2. https://github.com/moderntribe/tribe-libs/commit/eca41c95b84c0e6c38a3127dc24b836374d12a4b#diff-c4b17a47324748d068e45da15eb798ea25fcc161d1ffe2835aa6629e76a77dd3
3. https://github.com/moderntribe/tribe-libs/commit/a83f9013c07828880b46866f15175c568f705e7e#diff-c4b17a47324748d068e45da15eb798ea25fcc161d1ffe2835aa6629e76a77dd3
4. https://github.com/moderntribe/tribe-libs/commit/c34854b6fbf54658859a50c8c808856652ee5bf9#diff-c4b17a47324748d068e45da15eb798ea25fcc161d1ffe2835aa6629e76a77dd3
5. https://github.com/moderntribe/tribe-libs/commit/b29b16a91346187e6008555c5f8908456b4fd5d8#diff-c4b17a47324748d068e45da15eb798ea25fcc161d1ffe2835aa6629e76a77dd3

**Comment by defunctl** [comment]
@tarecord oh, add an item to the `Unreleased` changelog section as well, that gets updated to the latest release once the monorepo push goes out.

**Review by defunctl — CHANGES_REQUESTED** [review]
Add a changelog item and if you have time, there are a few other items that will also fail linting on the JS side of things

**Review by defunctl — APPROVED** [review]
Tested on a project with v2.0 coding standards and it works perfectly :rocket:

File: src/Generators/templates/component/source.js [hunk 1]
```diff
@@ -8,7 +8,7 @@
  * @description TODO: add a description to this file
  */
 
-import * as tools from 'utils/tools';
+// import * as tools from 'utils/tools';
```
Comment (dpellenwood) [inline src/Generators/templates/component/source.js:11]: Feel free to just delete this line. It's unnecessary in the boilerplate.

**Review by dpellenwood — APPROVED** [review]
This is wonderful. Just one minor tweak, then good to merge. Thanks so much @tarecord !

**Pull request merged**